### DISCUSSION TITLE
Palindromik Sayılar

### DISCUSSION THREAD

**Discussion by adasue** [sonsuzus/sonsuzus#332] [Soru Cevap]
Verilen tam sayı ` x `, eğer x **palindromikse** ` true ` döndür, ve aksi takdirde ` false `.

**Örnek 1:**
```
Giriş: x = 121
Çıktı: true
Açıklama: 121 hem soldan sağa ve hem de sağdan sola doğru 121 diye okunur.
```

**Örnek 2:**
```
Giriş: x = -121
Çıktı: false
Açıklama: Soldan sağa doğru, o -121 diye okunur. Sağdan sola doğru ise, 121- olur. Bu nedenle bu bir palindrom değildir.
```

**Örnek 3:**
```
Giriş: x = 10
Çıktı: false
Açıklama: Soldan sağa doğru 01 diye okunur. Bu yüzden o bir palindrom değildir.
```

**_Kısıtlamalar:_**
`-2³¹ <= x <= 2³¹ - 1`

**Takip eden:** Bu soruyu tam sayıyı bir diziye dönüştürmeden çözebilir misiniz?

Kaynakça:https://leetcode.com/problems/palindrome-number/

**Comment by sonsuzus** [comment]
Programlarken eksi sayıları doğrudan false yapabiliriz o zaman. 0 ın altında hiç bir sayı palindrom olamaz.
